modules/apps/change-tracking/change-tracking-store-test/src/testIntegration/java/com/liferay/change/tracking/store/internal/change/tracking/test/CTSContentTableReferenceDefinitionTest.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.change.tracking.store.internal.change.tracking.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.change.tracking.store.service.CTSContentLocalService;
+import com.liferay.change.tracking.test.util.BaseTableReferenceDefinitionTestCase;
+import com.liferay.document.library.kernel.store.Store;
+import com.liferay.petra.io.unsync.UnsyncByteArrayInputStream;
+import com.liferay.portal.kernel.model.change.tracking.CTModel;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Gislayne Vitorino
+ */
+@RunWith(Arquillian.class)
+public class CTSContentTableReferenceDefinitionTest
+	extends BaseTableReferenceDefinitionTestCase {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_fileName = "testFile";
+
+		_fileSystemStore.addFile(
+			TestPropsValues.getCompanyId(), 0, _fileName, Store.VERSION_DEFAULT,
+			new UnsyncByteArrayInputStream(_DATA_1));
+	}
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return _ctsContentLocalService.addCTSContent(
+			TestPropsValues.getCompanyId(), 0, _fileName, Store.VERSION_DEFAULT,
+			_STORE_TYPE, new UnsyncByteArrayInputStream(_DATA_1));
+	}
+
+	private static final byte[] _DATA_1 = "Data1 a".getBytes();
+
+	private static final String _STORE_TYPE =
+		"com.liferay.portal.store.file.system.FileSystemStore";
+
+	@Inject
+	private CTSContentLocalService _ctsContentLocalService;
+
+	private String _fileName;
```

Comment:
PR mostly looks good except for a minor style change that could be made here. You can declare `_fileName` as a private static final here to further clean up the setUp() method.

`private static final String _fileName = "testFile";`